jsonnet/thanos-ruler.jsonnet

```diff
@@ -305,6 +354,10 @@ local authorizationRole =
                   mountPath: '/etc/proxy/htpasswd',
                   name: 'secret-thanos-ruler-oauth-htpasswd',
                 },
+                {
+                  mountPath: '/etc/tls/grpc',
```

Comment:
do we really need to mount the GRPC secret in the oauth proxy container?

Reply:
I don't think so either, @pgier think this was in your initial commit, why do we need this?

Reply:
I think that's a mistake, it should only be mounted in the thanos-ruler container.

Reply:
agreed, this shouldn't be necessary here.